CHANGELOG-1.6.md

```diff
@@ -756,14 +756,14 @@ filename | sha256 hash
 
 ## Known Issues for v1.6.5
 
-* If you use the [GLBC Ingress Controller](https://github.com/kubernetes/ingress/tree/master/controllers/gce),
+* If you use the [GLBC Ingress Controller](https://github.com/kubernetes/ingress-nginx/tree/nginx-ingress-controller-0.9-beta.1/controllers/gce),
```

Comment:
I didn't expect nginx tags related to gce... cc @MrHohn or @bowei for what these should link to

Reply:
I guess it would be https://github.com/kubernetes/ingress-nginx/tree/0.9.0-beta.1/controllers/gce instead (gce ingress controller used to live in the same repo).

Reply:
@MrHohn Thanks